pkg/graph/resolvers/system_intake_test.go

```diff
@@ -222,6 +223,8 @@ func (s *ResolverSuite) TestGetRequesterUpdateEmailData() {
 	ctx := s.testConfigs.Context
 	store := s.testConfigs.Store
 
-	_, err := GetRequesterUpdateEmailData(ctx, store)
+	_, err := GetRequesterUpdateEmailData(ctx, store, func(ctx context.Context, strings []string) ([]*models.UserInfo, error) {
```

Comment:
Not directly part of this fix, but this unit test should probably be filled out more. I think this is effectively testing the empty state. 

Ideally, we should be testing that we return the correct EUA for each possible status.